zephyr/src/wpa_cli.c

```diff
@@ -12,15 +12,15 @@
 #include <stdlib.h>
 #include <ctype.h>
 
-#include <zephyr/zephyr.h>
+#include <zephyr/kernel.h>
 #include <zephyr/shell/shell.h>
 #include <zephyr/init.h>
 
-#include <net/net_if.h>
-#include <net/wifi_mgmt.h>
-#include <net/net_event.h>
+#include <zephyr/net/net_if.h>
+#include <zephyr/net/wifi_mgmt.h>
+#include <zephyr/net/net_event.h>
 
-#include <src/utils/common.h>
+#include <zephyr/src/utils/common.h>
```

Comment:
This is incorrect. There is no such file.

Reply:
Oops, a bit to late...

Reply:
oops..will raise another PR :-(.

Reply:
Its not even giving a build warning, may be I just do a force push.

Reply:
Wait, we need to use CONFIG_WPA_CLI=y